### PR TITLE
Chore: consolidate CCL release 3.3.x

### PR DESCRIPTION
**Pull request by mlenkeit**
This PR consolidates the changes for CCL 3.3.x in a separate branch, e.g. to leverage the OS-specific integration tests and to consolidate the UA review.

### CODE REVIEW

File: resources/i18n/de/strings.xml [hunk 1]
```diff
@@ -35,9 +81,9 @@
   <!-- XTXT: Admission State - OTHER Badge Text -->
   <string name="ADMISSION_STATE_OTHER_BADGE_TEXT">""</string>
   <!-- XTXT: Admission State - OTHER Card Body -->
-  <string name="ADMISSION_STATE_OTHER_LONG_TEXT">"Ihre Zertifikate erfüllen derzeit keine Regel."</string>
+  <string name="ADMISSION_STATE_OTHER_LONG_TEXT">"Der G-Status ist zur Zeit nicht relevant und wird daher nicht von der App ausgewiesen."</string>
   <!-- XTXT: Admission State - OTHER State Change Notification Text -->
-  <string name="ADMISSION_STATE_OTHER_SCN_TEXT">"Ihre Zertifikate erfüllen derzeit keine Regel."</string>
+  <string name="ADMISSION_STATE_OTHER_SCN_TEXT">"Der G-Status ist zur Zeit nicht relevant und wird daher nicht von der App ausgewiesen."</string>
```
Comment (mlenkeit) [inline resources/i18n/de/strings.xml:86]: These texts need double-checking, AFAIK we haven't used _G-Status_ so far in the app.

Comment (KathrinLuecke) [reply resources/i18n/de/strings.xml:86]: How about: "Zertifikate werden zur Zeit nicht geprüft und Ihr Status daher nicht von der App ausgewiesen."

**Review by KathrinLuecke — CHANGES_REQUESTED** [review]

File: lib/ccl/ccl-constants.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
-export const TC_PCR_VALIDITY_IN_HOURS = 72
-export const TC_RAT_VALIDITY_IN_HOURS = 48
+export const TC_PCR_VALIDITY_IN_HOURS = 24
+export const TC_RAT_VALIDITY_IN_HOURS = 24
```
Comment (Ein-Tim) [inline lib/ccl/ccl-constants.js:2]: Shouldn't this change be back ported to the apps "Receive test results" feature (e.g. show RAT as out of date after 24h)?

**Comment by Ein-Tim** [comment]
I have a question reg. https://github.com/corona-warn-app/cwa-app-ccl/pull/107, if the texts & the immunization state in the app changes to partial immunization only, will there be a booster recommendation notification which not only **recommends** the booster but clearly states that a booster is **necessary** now?

File: lib/ccl/ccl-constants.js (hunk 2 above)
Comment (mlenkeit) [reply lib/ccl/ccl-constants.js:2]: This is not yet decided.

Comment (Ein-Tim) [reply lib/ccl/ccl-constants.js:2]: @mlenkeit 

Ok, thanks, but it is in the talks?

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: lib/ccl/ccl-constants.js (hunk 2 above)
Comment (mlenkeit) [reply lib/ccl/ccl-constants.js:2]: Yes, but it will most likely stay as it is at the moment.

**Comment by mlenkeit** [comment]
> I have a question reg. #107, if the texts & the immunization state in the app changes to partial immunization only, will there be a booster recommendation notification which not only **recommends** the booster but clearly states that a booster is **necessary** now?

@Ein-Tim this is not planned at the moment.

**Comment by Ein-Tim** [comment]
@mlenkeit 

So the user will just loose their fully vaccinated status without further information in the app why?

I think more information should be given.

**Comment by MikeMcC399** [comment]
@Ein-Tim 
 
> So the user will just loose their fully vaccinated status without further information in the app why?
> 
> I think more information should be given.

This is more the responsibility of the new features text in the app e.g. for Android 2.27.x https://github.com/corona-warn-app/cwa-app-android/blob/release/2.27.x/Corona-Warn-App/src/main/res/values/release_info_strings.xml.

At the moment only the mask rule change is mentioned in the release info. It would be good for the release info to also explain that the 2G, 2G+, 3G and 3G+ status is removed and that the rules in general have been updated to correspond to current regulations.

Also the blog text for the release 2.27 would have the opportunity to explain about this.

This is all work-in-progress, so I would wait and see what develops.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I'll take your advice and look at it again in a week or so.

**Comment by Ein-Tim** [comment]
Okay, question answered with https://github.com/corona-warn-app/dcc-rule-translation/issues/10#issuecomment-1252418923.

**Pull request merged**